### ISSUE TITLE
Swap Fx function lock not working and I need to use two receivers?

### ISSUE THREAD

**Issue by W0nd3rLu5t3r** [pwr-Solaar/Solaar#1914]
Greetings. I am rather new in this space and I tried a little digging around, but if this doubles somewhere, please just point me to the right place.

So, I am using a K400 Plus keyboard and M720 Triathlon Mouse. The program is Solaar 1.1.1, running mostly flawlessly on Ubuntu 22.04.1 with XFCE (old Dell here).

The first problem is that I can't use only one receiver for the two devices. It doesn't matter to which one I pair them. I have to use both.

The second minor problem comes from the fact, that the **F** keys loose their normal behavior as F1-12 every time the system is restarted.  It doesn't matter what the settings in the Solaar interface _"Swap Fx function"_ appears to be, it is being ignored. Instead they default to the special functions and I have to use the Fn key +F# combination, or go to the Solaar interface, toggle the function on and off each time after a restart.

I hope I could find some directions here.

**Comment by pfps** [comment]
Solaar should allow you to pair both devices with one receiver.   However, you cannot pair a device with a receiver when it is paired and connected to another receiver.  So you generally have to unplug the receiver a device is paired with before pairing it with a different receiver.

Devices forget most of their settings when they go inactive even for a short time.   To keep changed settings you have to run the Solaar GUI continuously, e.g., arranging for Solaar to start when you log in.

Solaar version 1.1.1 is quite old.  You should consider using the current version of Solaar from one of the Solaar Ubuntu PPAs.

**Comment by W0nd3rLu5t3r** [comment]
Thank you very much for the observation, that my version is older. I am not sure how I installed the app when I did, but apparently something was off, since the repository wasn't in my list, thus I was not getting any auto updates. 

As of now I am with 1.1.7 and so far no problems. Everything is working fine. I will keep monitoring the Fn/Fx settings, but after the first restart my F1-F12 are right back there where I left them. I am one happy user right now :D

Thank you again! I wish you, and everyone involved in the Solaar manager many marry holidays ahead!
